en/components/general/ignite-ui-licensing.md

```diff
@@ -63,21 +62,42 @@ This will allow you to seamlessly use a mix of packages from the public npm regi
 >[!NOTE]
 > `npm` is disallowing the use of the `"@"` symbol inside your username as it is considered as being "not safe for the net". Because your username is actually the email that you use for your Infragistics account it always contains the symbol `"@"`. That's why you must escape this limitation by replacing the `"@"` symbol with `"!!"` (two exclamation marks). For example, if your username is `"username@example.com"` when asked about your username you should provide the following input: `"username!!example.com"`.
 
-#### Now, to log in to our private feed using npm, run the adduser command and specify a user account and password:
+### Now, to log in to our private feed using npm
 
-```cmd
-npm adduser --registry=https://packages.infragistics.com/npm/js-licensed/ --scope=@infragistics --always-auth
-```
->[!NOTE]
-> If you are using `npm` version 9.0.0 or higher you need to set `--auth-type=legacy`
+#### npm version 9.4+
+<pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
+npm config set @infragistics:registry https://packages.infragistics.com/npm/js-licensed/
 
-```cmd
-npm adduser --registry=https://packages.infragistics.com/npm/js-licensed/ --scope=@infragistics --always-auth --auth-type=legacy
-```
+npm config set //packages.infragistics.com/npm/js-licensed/:username=${username}
```

Comment:
2 minor nits - what's with the `<pre>` tags? The `cmd` tagged code block was fine, alternatively pick something else (bash, shell, whatever is supported), just not this inline style thing. Also wouldn't use possibly valid variable syntax here with the presumption users would magically get it. Instead, use something that clearly indicates these values are to be replaced - a common for that would be angle brackets, i.e. `username=<username>`, though likely not the only option (can't tell if we have a precedent already)

Reply:
Actually, @zdrawku the username should be irrelevant with `_auth` (at least with newer versions, finally), so we might be able to skip it?

Reply:
I will resolve this tomorrow, for the time being please approve